test/utils/utf8_demo.cpp

```diff
@@ -0,0 +1,222 @@
+// The content of the "UTF-8 encoded sample plain-text file" by Markus Kuhn.
+// SPDX-License-Identifier: CC-BY-4.0
+#include "utf8_demo.hpp"
+
+namespace fizzy::test
+{
+const std::string_view utf8_demo =
```

Comment:
Isn't string_view a reference?

Reply:
It is, but reference to a literal is fine.